.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,81 @@
+name: Docker Image CD
+
+on:
+  workflow_dispatch:
+    inputs:
+      bin_tag:
```

Comment:
This is only executed when there is a new tag, right?

Reply:
workflow_dispatch is a manual trigger, I don't know where can I receive the state of the binaries to dispatch the pipeline 🤔

Reply:
On other repos we do this:
```
on:
  release:
    types:
      - published
``` 
and then we get the value of the tag like this:
```
    - id: tag
      name: Get the tag
      run: echo ::set-output name=TAG::${GITHUB_REF#refs/tags/}

    - name: "Push"
      run: |
        export TAG=${{ steps.tag.outputs.TAG }}
        make push
```

Reply:
Ohhh! Ok!, I didn't notice your uploading the bin as a release in the repo. That's perfect, I will add the same trigger and removing the workflow_dispatch.